libs/core/src/lib/analytics/tracking.service.ts

```diff
@@ -90,15 +91,17 @@ export class STFTrackingService {
     return this.postHogService.getEnabledFeatures();
   }
 
-  getStatusAlert(): Observable<string> {
+  getStatusMessage(alert: boolean): Observable<string> {
```

Comment:
ok now I understand the `true`/`false` call. It would be easier to understand and easier to extend by using a type instead. like `'status' | 'announce'`.